wptool/wptool.cs

```diff
@@ -0,0 +1,232 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using Microsoft.SmartDevice.Connectivity.Interface;
+using Microsoft.SmartDevice.MultiTargeting.Connectivity;
+using System.Net;
+
+namespace wptool
+{
+	class wptool
+	{
+		static int Main(string[] args) {
+			if (args.Length == 0) {
+				return showHelp();
+			}
+
+			string command = null;
+			string wpsdk = null;
+			string udid = null;
+            string appid = null;
```

Comment:
The indention in this file is pretty bad. Could you please do a quick scrub to ensure everything lines up? Thank you!